notebook/templates/page.html

```diff
@@ -24,25 +29,25 @@
           {% endif %}
           baseUrl: '{{static_url("", include_version=False)}}',
           paths: {
-          {% if ignore_minified_js %}
-            'auth/js/main': 'auth/js/main',
-          {% else %}
-            'auth/js/main': 'auth/js/main.min',
-          {% endif %}
+            'auth/js/main': 'auth/js/built/main.min',
             custom : '{{ base_url }}custom',
             nbextensions : '{{ base_url }}nbextensions',
             widgets : '{{ base_url }}deprecatedwidgets',
             kernelspecs : '{{ base_url }}kernelspecs',
             underscore : 'components/underscore/underscore-min',
             backbone : 'components/backbone/backbone-min',
-            jquery: 'components/jquery/jquery.min',
-            bootstrap: 'components/bootstrap/js/bootstrap.min',
-            bootstraptour: 'components/bootstrap-tour/build/js/bootstrap-tour.min',
-            'jquery-ui': 'components/jquery-ui/ui/minified/jquery-ui.min',
             moment: 'components/moment/moment',
             codemirror: 'components/codemirror',
             termjs: 'components/term.js/src/term',
-            typeahead: 'components/jquery-typeahead/dist/jquery.typeahead'
+
+            // Anything jQuery related is loaded globally because it will
+            // pollute the global object regardless and we want to avoid one
+            // jQuery thinger from clompering another.
+            jquery: '//deprecated//',
+            bootstrap: '//deprecated//',
+            bootstraptour: '//deprecated//',
+            'jquery-ui': '//deprecated//',
+            typeahead: '//deprecated//'
```

Comment:
Yeah, I agree requiring from the global namespace is bad, unfortunately jquery things insist on being global.

But yeah, I can totally make the requirements continue to work, and just console.warn a deprecation warning